riak/datadog_checks/riak/riak.py

```diff
@@ -224,42 +224,41 @@ def __init__(self, name, init_config, agentConfig, instances=None):
                 self.keys.append(m + "_" + k)
 
         self.prev_coord_redirs_total = -1
-
-    def check(self, instance):
-        url = instance['url']
+        self.url = self.instance['url']
         default_timeout = self.init_config.get('default_timeout', 5)
```

Comment:
```suggestion
        default_timeout = float(self.init_config.get('default_timeout', 5))
```

Just catching this at the same time, we don't update riak really often.